sources/accessible-name-and-description.ts

```diff
@@ -306,11 +307,27 @@ function getLabels(element: Element): HTMLLabelElement[] | null {
 	});
 }
 
+/**
+ * Gets the contents of a slot used for computing the accname
+ * @param slot
+ */
+function getSlotContents(slot: HTMLSlotElement): Node[] {
+	// Computing the accessible name for elements containing slots
+	// is not currently defined in the spec. This implementation
+	// reflects current browser behavior.
```

Comment:
nit: With what browser did you verify the changes? I guess Chrome but for future changes it'd be nice to have this reference. Especially if browser behavior changes.

Reply:
I tested Firefox/NVDA and iOS Safari/VoiceOver. I'll update the comment with the browser/SR combos and versions.

Reply:
Code comment has been updated with the above. Please resolve if you're satisfied with the change.